cql3/restrictions/statement_restrictions.cc

```diff
@@ -708,6 +708,9 @@ bool single_column_restriction::slice::is_satisfied_by(const schema& schema,
 }
 
 bool single_column_restriction::slice::is_satisfied_by(bytes_view data, const query_options& options) const {
+    if (data.empty()) {
+        return false; // NULL is not contained in any range
+    }
```

Comment:
NULLs are signified by a bytes_view_opt being disengaged. An empty string is not null.

Reply:
Ah, I see. So it's much harder, I'm closing this one.